### DISCUSSION TITLE
Rename/reorder only?

### DISCUSSION THREAD

**Discussion by thepwrtank18** [enzo1982/freac#441] [Help & Support]
Is there a way to not convert anything to another format, but just rename the files and folders to what they are if they were converted? (ex: "[Album] Alstroemeria Records – Bad Apple!! feat.nomico 10th Anniversary PHASE2 [FLAC]" just becomes "Alstroemeria Records - Bad Apple!! feat.nomico 10th Anniversary PHASE2")

**Comment by Lee-Carre** [comment]
> Is there a way to not convert anything to another format, but just rename the files and folders to what they are if they were converted?

I'm not sure, within Fre:AC. However, this is possible with [MusicBrainz Picard](//picard.musicbrainz.org/); renaming/organising files to conform to a pattern is a sub-feature, with a scripting mini-language and all.
